pkgs/by-name/ho/hoppscotch/package.nix

```diff
@@ -34,41 +39,56 @@ let
     '';
     homepage = "https://hoppscotch.com";
     downloadPage = "https://hoppscotch.com/downloads";
-    changelog = "https://hoppscotch.com/changelog";
-    license = licenses.mit;
-    maintainers = with maintainers; [ DataHearth ];
+    changelog = "https://github.com/hoppscotch/hoppscotch/releases/tag/2024.8.1";
```

Comment:
The version here should have been dynamic with some string interpolation on the `version` binding.

Reply:
https://github.com/NixOS/nixpkgs/pull/343315